CakePHP/Tests/Commenting/DocBlockAlignmentUnitTest.1.inc

```diff
@@ -23,8 +23,9 @@ class Foo extends Bar
  *
  * @return void
  */
-    public function dumpThoughts()
+    public function dumpThoughts($thoughts)
     {
+        /** @var array $thoughts */
```

Comment:
It would indeed, and neglecting to add that made me miss an error in how auto-fixing worked for single-line doc comments. Thanks!